FlowCryptCommon/Logger.swift

```diff
@@ -92,7 +92,7 @@ public struct Logger {
 
     private func log(
         _ level: Logger.Level,
-        _ message: @autoclosure () -> String,
+        _ message: String,
```

Comment:
This was causing logs to get written with a huge delay, or sometimes after certain unrelated actions completed, making it hard to debug